cmsdb/campaigns/run3_2023_postBPix_nano_v12/data.py

```diff
@@ -0,0 +1,84 @@
+# coding: utf-8
+
+"""
+CMS datasets from the 2023 postBPix data-taking campaign
+"""
+
+import cmsdb.processes as procs
+from cmsdb.campaigns.run3_2023_postBPix_nano_v12 import campaign_run3_2023_postBPix_nano_v12 as cpn
+
+# Add era E also?
```

Comment:
Era E is not included [here](https://twiki.cern.ch/twiki/bin/viewauth/CMS/PdmVRun3Analysis#DATA_AN1), o I suppose it is not necessary